docs/02-app/01-building-your-application/11-upgrading/05-from-create-react-app.mdx

```diff
@@ -141,7 +99,7 @@ export default function RootLayout({ children }) {
 
 > **Good to know**: `.js`, `.jsx`, or `.tsx` extensions can be used for Layout files.
 
-Copy the content of your `index.html` file into the previously created `<RootLayout>` component while replacing the `body.div#root` and `body.script` tags with `<div id="root">{children}</div>`:
+Copy the content of your `index.html` file into the previously created `<RootLayout>` component while replacing the `body.div#root` and `body.noscript` tags with `<div id="root">{children}</div>`:
```

Comment:
The `body.script` tag doesn't exist on a CRA setup by default. [There's instead a `noscript` tag](https://github.com/IGassmann/cra-to-nextjs/blob/14d4292a3bafca457d6c576b1c0811fdb546ffb6/public/index.html#L30) that needs to be removed.